utils/config.go

```diff
@@ -602,10 +602,12 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	props["DataRetentionMessageRetentionDays"] = "0"
 	props["DataRetentionEnableFileDeletion"] = "false"
 	props["DataRetentionFileRetentionDays"] = "0"
+	props["RunJobs"] = "false"
 
 	if license != nil {
 		props["ExperimentalTownSquareIsReadOnly"] = strconv.FormatBool(*c.TeamSettings.ExperimentalTownSquareIsReadOnly)
 		props["ExperimentalEnableAuthenticationTransfer"] = strconv.FormatBool(*c.ServiceSettings.ExperimentalEnableAuthenticationTransfer)
+		props["RunJobs"] = strconv.FormatBool(*c.JobSettings.RunJobs)
```

Comment:
This isn't an enterprise only config so I think it should always be exposed, even though we don't need it in TE for this particular case.

Reply:
Yes I agree.